code/modules/medical/cloner_defects.dm

```diff
@@ -238,36 +258,56 @@ ABSTRACT_TYPE(/datum/cloner_defect/missing_organ)
 		qdel(src.owner.organHolder.organ_list[data["organ_string"]])
 
 /datum/cloner_defect/missing_organ/minor
-	severity = CLONER_DEFECT_SEVERITY_MINOR
+	severity = CLONER_DEFECT_SEVERITY_MAJOR
 
 	init()
-		src.data = list("organ_string" = pick("tail", "left_kidney", "right_kidney", "stomach", "intestines", "appendix"))
+		src.data = list("organ_string" = pick("tail", "left_kidney", "right_kidney", "stomach", "intestines", "appendix", "liver",))
 
 /datum/cloner_defect/missing_organ/major
 	severity = CLONER_DEFECT_SEVERITY_MAJOR
 
 	init()
-		src.data = list("organ_string" = pick("liver", "left_eye", "right_eye", "left_lung", "right_lung")) // no skull, head, brain; instant death sucks. also removed heart
+		src.data = list("organ_string" = pick("left_eye", "right_eye", "left_lung", "right_lung")) // no skull, head, brain; instant death sucks. also removed heart
```

Comment:
the minor organ missing stuff is all things that are either relatively or entirely harmless, and was classified as minor as such
making it major and putting liver in there (which will kill you via tox damage) is a strange choice

Reply:
This was by accident, actually.